Sources/Sentry/SentrySDK.m

```diff
@@ -350,7 +351,7 @@ + (void)installIntegrations
                 integrationName);
             continue;
         }
-        id<SentryIntegrationProtocol> integrationInstance = [[integrationClass alloc] init];
```

Comment:
This was an especially important change. Because the init is defined in SentryBaseIntegration and not SentryIntegrationProtocol, when I changed it from SentryBaseIntegration.init to SentryBaseIntegration.initWithCrashWrapper:, this line didn't break compilation. It would have gone on to crash at runtime. This is an unsafe code sector in general, so this makes it a little better.